example-app/src/main/res/layout/activity_main.xml

```diff
@@ -60,6 +60,14 @@
             android:checked="false"
             android:text="@string/web_view_debug"
             style="@style/CheckBoxText" />
+        <CheckBox
```

Comment:
Let's switch the different checkbox sizes to a drop down instead.
Size: [drop-down-selector] (normal, compact, invisible), defaults to `normal`.

instead of current checkboxxes.
![image](https://user-images.githubusercontent.com/11096226/172608953-71bb424e-2f10-46ac-a9f6-4be118d30f78.png)

Reply:
As far as I understand size=normal means that image picker UI will be show, correct me if I'm wrong

Reply:
Done